pkg/kube/client.go

```diff
@@ -390,7 +390,7 @@ func updateResource(c *Client, target *resource.Info, currentObj runtime.Object,
 		return fmt.Errorf("failed to create patch: %s", err)
 	}
 	if patch == nil {
-		log.Printf("Looks like there are no changes for %s", target.Name)
+		log.Printf("Looks like there are no changes for %s %s", target.Mapping.GroupVersionKind.Kind, target.Name)
```

Comment:
- [ ] At line 155 above, we quote the object name, but here we don't.  
  Make that treatment consistent. I recommend using the `%q` format verb for the object name.